v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/templates/PipelineController.java

```diff
@@ -72,12 +74,25 @@ static PipelineResult executeSingleInstanceMigration(
     List<String> orderedSpTables = ddl.getTablesOrderedByReference();
 
     Map<String, PCollection<Void>> outputs = new HashMap<>();
-
+    // This list will contain the final list of tables that actually get migrated, which will be the
+    // intersection of Spanner and source tables.
+    List<String> finalTablesToMigrate = new ArrayList<>();
     for (String spTable : orderedSpTables) {
       String srcTable = schemaMapper.getSourceTableName("", spTable);
       if (!tablesToMigrateSet.contains(srcTable)) {
         continue;
       }
+      finalTablesToMigrate.add(spTable);
+    }
+    LOG.info("Final list of Spanner tables that will be populated: ", finalTablesToMigrate);
```

Comment:
Please add the number of tables here. Is this log printing considering there is no {} ?